packages/gateway/src/features/user/root-resolvers.ts

```diff
@@ -409,33 +388,26 @@ export const resolvers: GQLResolver = {
       })
 
       if (res.status !== 200) {
-        return await Promise.reject(
-          new Error(
-            "Something went wrong on user-mgnt service. Couldn't change user phone number"
-          )
+        throw new Error(
+          "Something went wrong on user-mgnt service. Couldn't change user phone number"
         )
       }
-      return true
```

Comment:
@noah-brunate removing this line here might break things as people could be expecting a boolean return value from the API.
This change is also the reason why you had to change the expectations in the test file. Let's bring this back please